src/db.ts

```diff
@@ -1,10 +1,10 @@
-import { Connection, createConnection } from 'typeorm';
+import { DataSource } from 'typeorm';
 import { dbUrl } from './env';
 import { Rep } from './entities/Rep';
 import { HelpThread } from './entities/HelpThread';
 import { Snippet } from './entities/Snippet';
 
-let db: Connection | undefined;
+let db: DataSource | undefined;
```

Comment:
Connection was deprecated in favor of DataSource because the name made more sense, so... fine.